File: Coding/Python/sum_3_5.py
# Program that asks the user for a number n and then will sum all the numbers from 1 to that number that have factors of 3 or 5

print "Hello user please enter the number to sum to: "

n = raw_input()
sum_nums = 0

for i in range (0, int(n)+1):
    if (i%3 == 0) or (i%5 == 0):
        sum_nums += i
    
print "The sum of all the numbers from 1 to " + n + " that have factors of 3 or 5  is " + str(sum_nums) + "."

File: Coding/Perl/sum_3_5.pl
# Program that will ask a user for a number(n) and then sum the numbers from 1 to n

print "Hello user please enter the number n to sum from 1 to n: ";
chomp($n=<>);

$sum = 0;

for($i=1;$i<=$n;$i++){
    if($i%3 == 0 or $i%5 == 0 ){
        $sum += $i;
    }
}

print "The sum of the numbers 1 to n that have factors of the numbers 3 or 5 is $sum."